### ISSUE TITLE
No results generated

### ISSUE THREAD

**Issue by zwhyyyy** [JWLiang007/PFF#2]
Hello, I followed your tutorial step by step. During my operation, the words "data/Celeb-DF-v2/Celeb-synthesis/videos/id30_id4_0001.mp4 Done!
data/Celeb-DF-v2/Celeb-synthesis/videos/id25_id19_0008.mp4 Done!" appeared, but why are there no pictures generated in my folder? Looking forward to your reply. Thank you

**Comment by JWLiang007** [comment]
Hi. 

During the testing process, the code extracts images from videos and saves them as batches in NumPy format. You should find paths like data/FaceForensics++/manipulated_sequences/Deepfakes/c23/same_advt_gen_i_e_sharpen_5/000_003/croppedfaces.npy in your data folder. To view the images, locate these NumPy files, read them with np.load(), and convert them into image format using PIL or OpenCV. Hope this helps!

By the way, the preprocessing may be time-consuming; you may need to wait a while until it's done.

**Comment by zwhyyyy** [comment]
Thank you for your answer. I would like to ask, do .npy files need to be converted into .png format before training? I am a novice and my question may be a bit simple. Thank you.

**Comment by JWLiang007** [comment]
No, you don’t need to convert the .npy files into .png format before training. The aforementioned conversion from numpy array to images is only for visualization purposes.

**Comment by zwhyyyy** [comment]
OK, I understand. Thank you.
